skills/golem-powers/agada-bench/scripts/run-agada.sh

```
@@ -0,0 +1,271 @@
+  local PRIMARY_JUDGES=""
+  local SHADOW_JUDGE=""
+  DRY_RUN="${DRY_RUN:-false}"
+  local FORCE="false"
+
+  while [[ $# -gt 0 ]]; do
+    case "$1" in
+      --session) SESSION="$2"; shift 2 ;;
+      --domain) DOMAIN="$2"; shift 2 ;;
+      --output) OUTPUT="$2"; shift 2 ;;
+      --judges) JUDGES="$2"; shift 2 ;;
+      --rubric-version) RUBRIC_VERSION="$2"; shift 2 ;;
```

Shellcheck (0.11.0)

[warning] RUBRIC_VERSION appears unused. Verify use (or export if used externally).

(SC2034)